test/unit/rate-limit-test.spec.ts

```diff
@@ -83,38 +83,6 @@ it('Retry on 5** - multiple errors', async () => {
   expect(response.data).toEqual('works #2')
 })
 
-it('Retry on 5** - multiple errors - reach/exceed limit', async () => {
```

Comment:
Should we add a test for the attempts limit reached case?

Reply:
There is already `Should Fail if it hits maxRetries` https://github.com/contentful/contentful-sdk-core/pull/239/files#diff-ba11934f531b04b253b7d2a82ee29cef9a27e4bcbebd90ea0bb46a68ebaced58R118, which is now the same for 429, 5XX or network level errors so this case should be already covered.